### DISCUSSION TITLE
How to set resources.limits.request.memory in jaeger-agent sidecar ?

### DISCUSSION THREAD

**Discussion by frank-cq** [jaegertracing/jaeger-operator#1832] [Q&A] [answered]
I use annotation sidecar.jaegertracing.io/inject to enable jaeger-client autoinject in my Deployment. However the hpa told me it was failed to get the memory request of the jaeger-client while missing resources.limits.request.memory . Where can I set this configuration without modifying my deployment directly ?

**Comment by pavolloffay** [comment, marked as answer]
The agent spec https://pkg.go.dev/github.com/jaegertracing/jaeger-operator/apis/v1#JaegerAgentSpec embeds the `JaegerCommonSpec` that allows setting resources for the agent.

**Reply by frank-cq** [reply]
Don't familiar with this. I'll try it. Thank you.
